cypress/pageobjects/virtualmachine.po.ts

```diff
@@ -140,13 +143,18 @@ export class VmsPage {
   }
 
   public init() {
-    cy.intercept('GET', `/v1/harvester/harvesterhci.io.virtualmachineimages`).as('getImages');
-
     image.goToList();
-    
-    cy.wait('@getImages').then(res => {
-      expect(res.response?.statusCode).to.equal(200);
-      const images = res.response?.body?.data || []
+    cy.request({
+      url: 'v1/harvester/harvesterhci.io.virtualmachineimages',
```

Comment:
This should be saved as a constant in the constants file in case we need to change it easily later.

Reply:
Done